docs/knowledge_base/demos/Production planning.md

```diff
@@ -0,0 +1,49 @@
+Production planning is a pivotal aspect of manufacturing, ensuring that resources are allocated efficiently to meet demand while minimizing costs. Taipy's Production Planning demo offers a comprehensive solution for optimizing production and cost management. In this demo, we'll delve into this 5-page application that addresses the challenges of production planning.

```

Comment:
An hyperlink of 'Production planning' to a relevant page would be nice (like https://en.wikipedia.org/wiki/Production_planning).
"Taipy's Production Planning demo" -> "The Taipy Production Planning demo"
"5-page" is useless at this point.